pkg/controllers/disruption/validation.go

```diff
@@ -73,7 +76,7 @@ func (v *Validation) IsValid(ctx context.Context, cmd Command) (bool, error) {
 		case <-v.clock.After(waitDuration):
 		}
 	}
-	validationCandidates, err := GetCandidates(ctx, v.cluster, v.kubeClient, v.recorder, v.clock, v.cloudProvider, v.ShouldDisrupt)
+	validationCandidates, err := GetCandidates(ctx, v.cluster, v.kubeClient, v.recorder, v.clock, v.cloudProvider, v.ShouldDisrupt, v.queue)
```

Comment:
Just a question: Does the queueing mechanism here have any effect on our ability to validate? Just briefly thinking about it, I think we are still aright because nodes that we add to the queue are removed from the candidate pool synchronously so I think that these validation candidates should be consistent, even during parallel operations

Reply:
Yeah if anything we should be fail closed here